### ISSUE TITLE
Error message

### ISSUE THREAD

**Issue by peterbata** [CaramelFur/Picsur#98]
Let's see if I can remember what the error was: Something along the lines of:

This server is not compatible with the version of this client. Then I am asked to choose between BACC or IGNORE.

Is this anything that I should worry about.

Sys Info: Unraid v.6.12.13
Picsur installed from community apps

Thank you

**Comment by CaramelFur** [comment]
Nope, nothing to worry about, it's a bug in the checking system. It has been removed in the current version, but what you saw was a remnant from the previous version. From this point on this message should no longer appear.

**Comment by peterbata** [comment]
> Nope, nothing to worry about, it's a bug in the checking system. It has been removed in the current version, but what you saw was a remnant from the previous version. From this point on this message should no longer appear.

Thank you for the prompt response. Much appreciated.
